src/channels/components/ChannelForm/ChannelForm.tsx

```diff
@@ -212,6 +216,13 @@ export const ChannelForm: React.FC<ChannelFormProps> = ({
           hasError={!!formErrors.defaultTransactionFlowStrategy}
           disabled={disabled}
         />
+        <Box />
```

Comment:
questtion: do we need this empty box?

Reply:
![CleanShot 2024-10-04 at 11 03 49](https://github.com/user-attachments/assets/8e22ae31-7da1-4f55-98dc-9bfc93ecd988)

container is written incorrectly, so Box seems to be used to push the component to the new line. This should be refactored but its out of the scope of my changes